Dockerfile

```diff
@@ -1,7 +1,6 @@
+# -*- mode: dockerfile-mode; docker-image-name: "tailscale-dev"; -*-
```

Comment:
This makes `C-c C-b` use the image name `tailscale-dev` without having to enter it in over and over.